### PR TITLE
Issue 1 user can create an entry  - Colin Macrae

### PR DESCRIPTION
**Pull request by colin-macrae**

### CODE REVIEW

**Comment by colin-macrae** [comment]
changed border focus color to purple-ish color

**Comment by colin-macrae** [comment]
changed border focused to purple.

File: css/layout.css [hunk 1]
```diff
@@ -23,12 +23,22 @@
...
+    display: flex;
+  }
+
+  .row1-col2 {
     width: 50%;
+    margin: 0 0 0 15px;
+  }
+
+  .image {
+    width: 50%;
```
Comment (shawnkost) [inline css/layout.css]: You should move this to `styles.css`. The only CSS that I would apply here would be your container / row / col classes or utility classes

File: css/layout.css [hunk 2]
```diff
@@ -23,12 +23,22 @@
...
+
+    /* height: 600px; */
```
Comment (shawnkost) [inline css/layout.css]: You should be deleting all commented out or old code

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+  margin: 12px 0 0;
+}
+
+button {
+  margin: 10px 0 0;
+  background-color: rgb(103, 43, 135);
+  width: 80px;
+  height: 40px;
+
+  /* text-align: center */
```
Comment (shawnkost) [inline css/styles.css]: You should delete all commented out or old code

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+  border: none;
+  border-radius: 3px;
+
+  /* height:  */
```
Comment (shawnkost) [inline css/styles.css]: You should delete all commented out or old code

File: index.html [hunk 5]
```diff
@@ -15,16 +15,44 @@
...
   <main class="container">
-    <div data-view="entry-form"></div>
+    <form data-view="entry-form">
+      <h1>New Entry</h1>
+
+      <div class="row-one">
+        <img src="images/placeholder-image-square.jpg" class="image">
+
+        <div class="row1-col2">
+          <div class="title-label column-half"><label class="title" >Title</label></div>
```
Comment (shawnkost) [inline index.html]: All `label` elements should have a `for` attribute that matches the `id` of an `input`

File: index.html [hunk 6]
```diff
@@ -15,16 +15,44 @@
...
+          <input type="text" name="title" class="column-half title-label" required="required"/>
```
Comment (shawnkost) [inline index.html]: Give this an `id`

File: index.html [hunk 7]
```diff
@@ -15,16 +15,44 @@
...
+
+          <div class="photo-label column-half"><label for="##" class="">Photo URL</label></div>
```
Comment (shawnkost) [inline index.html]: All `label` elements should have a `for` attribute that matches the `id` of an `input`

File: index.html [hunk 8]
```diff
@@ -15,16 +15,44 @@
...
+          <input type="url" name="photo" class="photo-url column-half photo-label" required="required"/>
```
Comment (shawnkost) [inline index.html]: Give this an `id`

**Review by shawnkost — CHANGES_REQUESTED** [review]
Looking good Colin! Just a few changes requested here!

**Review by shawnkost — APPROVED** [review]
Nicely done 👍🏻

**Pull request merged**